Makefile

```diff
@@ -17,5 +17,5 @@ download-cosmos-proto:
 	mkdir -p proto/gogoproto
 	curl https://raw.githubusercontent.com/cosmos/cosmos-sdk/${cosmos-version}/third_party/proto/gogoproto/gogo.proto -o proto/gogoproto/gogo.proto
 
-gen-proto-ts: download-cosmos-proto install-gen-tool
+gen-proto-ts: download-cosmos-proto
```

Comment:
I wonder if `download-cosmos-proto` will also cause issues if we are not careful with what folders are mapped within Docker.

Reply:
I assume that `protoDirs` is set to `../proto` and `outPath` is set to `../client/src/types/generated`, so as long as the working dir is `/checkers`, I guess there wouldn't be issue when executing `npx telescope transpile --config .telescope.json` inside `scripts` folder.